package.json

```diff
@@ -10,13 +10,13 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^14.0.0",
-    "@rollup/plugin-node-resolve": "^8.4.0",
-    "rollup": "^1.16.2",
-    "rollup-plugin-postcss": "^2.0.3",
-    "rollup-plugin-serve": "^1.0.1",
-    "rollup-plugin-svelte": "^5.1.0",
-    "svelte": "^3.24.1"
+    "@rollup/plugin-commonjs": "^15.1.0",
+    "@rollup/plugin-node-resolve": "^9.0.0",
+    "rollup": "^2.29.0",
```

Comment:
Upgrading all the libraries as part of a PR for a feature isn't advisable. It's a great idea to do this, but it should be in an isolated PR on its own.